src/libtrx/game/objects/traps/rolling_ball.c

```diff
@@ -28,15 +28,12 @@ static void M_Roll(ITEM *const item)
     item->fall_speed = 0;
     item->pos.y = item->floor;
 
-    if (g_TRVersion != 2) {
```

Comment:
The boulders in TR1 constantly restart the SFX when they're active. I think this guard might be needed still.

Reply:
Restored. Thanks. I hate tr1 sound flags…